css/layouts/search.scss

```diff
@@ -0,0 +1,63 @@
+// styling for search popover from header
+
+.header-right {
+    .unified-search__input-wrapper label {
+        display: none;
+    }
+    
+    .unified-search__input-wrapper ~ .empty-content {
+        display: none;
+    }
+    
+    .empty-content {
+        &__icon > .material-design-icon.magnify-icon{
+            display: none;
+        }
+    }
+
+    .header-menu {
+        &__carret {
+            display: none;
+        }
+    }
+}
+
+#unified-search__input {
+    border: 2px solid black;
+    border-radius: 5px;
+    padding-top: 1.5rem;
+    padding-bottom: 1.5rem;
+    padding-left: 1rem;
+}
+
+
+.unified-search__result-icon.unified-search__result-icon--no-preview {
+    &.icon-folder {
+        background-size: 64px;
+        background-image: url('../img/filetypes/folder.svg');
+    }
+}
+
+.unified-search__form-reset.icon-close {
+    background-image: url('../img/actions/close.svg');
+    opacity: 1 !important;
+    margin: 12px 3px !important;
+}
+
+
+.unified-search__results-header {
+    color: #000000 !important;
+    font-size: 16 px !important;
+}
+
+.unified-search__result {
+    border: none !important;
+
+    &:active,
+	&:hover,
+	&:focus {
+		background-color: #ededed !important;
```

Comment:
we have a global variable for this color --nmc-color-background-hover, it is declared in nmcdefault.scss file